build.js

```diff
@@ -559,9 +563,43 @@ module.exports = function (gulpWrapper, ctx) {
                 });
                 
             }
-		})
+        })
         cb();
     });
+
+    /**
+     * Generates a PO file from all i18n/*.ts files and save it on baseDir
+     */
+    gulp.task('i18n-generate-po', function() {
+        return gulp
+            .src([
+                path.join(ctx.baseDir, "**/i18n/*.ts"),
+                "!**/*.d.ts"
+            ], { cwd: ctx.baseDir })
+            .pipe(pluginI18nTransform({
+                base: ctx.baseDir,
+                languages: i18n.supportedCultures,
+                dest: "pot"
+            }))
+            .pipe(gulp.dest(ctx.baseDir));
+    });
+
+    /**
+     * Restores i18n/*.ts files from previously .po generated files
+     */
+    gulp.task('i18n-restore-po', function() {
```

Comment:
Actually, the 18-transform repo address these tasks as Transform TS -> PO and Transform PO -> TS